src/style/card.css

```diff
@@ -0,0 +1,54 @@
+.product__card {
+  display: flex;
+  flex-direction: column;
+  font-weight: 500;
+  padding: 33px 16px 16px;
+  border: 1px solid var(--color--dark-white);
+  background-color: var(--color--white);
+}
+  
+.product__container {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 36px;
+  background-position: center;
+}
+  
+.product__containre-image {
+  display: flex;
+  width: 160px;
+  height: 134px;
+}
+  
+.product__card-title {
+  flex-direction: column;
+  justify-content: flex-end;
```

Comment:
flex-direction, justify-content don't work without display:flex. Please remove this two rules

Reply:
i did